pkg/nfs/nodeserver.go

```diff
@@ -73,6 +73,10 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	if err := os.Chmod(targetPath, os.FileMode(ns.Driver.perm)); err != nil {
```

Comment:
it should be possible to opt-out from the chmod thing entirely and hence `ns.Driver.Perm` should default to `nil` may be?

Reply:
agree. now if the value is not set, we omit the chmoding.